frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -167,11 +157,11 @@ func (a *CreateCVDAction) launchWithCanonicalConfig(op apiv1.Operation) (*apiv1.
 	opts := cvd.CommandOpts{
 		Timeout: a.cvdStartTimeout,
 	}
-	cmd := cvd.NewCommand(a.execContext, a.paths.CVDBin(), args, opts)
+	cmd := cvd.NewCommand(a.execContext, cvdBin, args, opts)
```

Comment:
nit: With several files in this package it's not immediately obvious where `cvdBin` is declared and whether it's a constant or global variable. Consider moving all constants to a constants.go file or use `cvdBin` only from the file it's declared in.

Reply:
moved the constants to the `cvd` package